stylecop.json

```diff
@@ -0,0 +1,27 @@
+﻿{
+  // ACTION REQUIRED: This file was automatically added to your project, but it
+  // will not take effect until additional steps are taken to enable it. See the
+  // following page for additional information:
+  //
+  // https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/EnableConfiguration.md
```

Comment:
If the "additional steps" have been added, should this be removed / updated?

Reply:
Ahh good spot. No I think we keep this in as I have raised a bug for SonarCloud and I think this is because the "additional steps" have yet to be carried out. Lets leave the removal of this until we have addressed that?

Reply:
https://skillsfundingagency.atlassian.net/browse/DFCC-1291